mmv1/products/bigqueryanalyticshub/Subscription.yaml

```diff
@@ -0,0 +1,150 @@
+    vars:
+      data_exchange_id: 'my_data_exchange'
+      listing_id: 'my_listing'
+      destination_dataset_id: 'destination_dataset'
+      description: 'example data exchange'
+parameters:
+  - name: 'data_exchange_id'
+    type: String
+    description: |-
+      The ID of the data exchange. Must contain only Unicode letters, numbers (0-9), underscores (_). Should not use characters that require URL-escaping, or characters outside of ASCII, spaces.
+    url_param_only: true
+    required: true
+  - name: 'listing_id'
+    type: String
+    description: |-
+      The ID of the listing. Must contain only Unicode letters, numbers (0-9), underscores (_). Should not use characters that require URL-escaping, or characters outside of ASCII, spaces.
+    url_param_only: true
+    required: true
+  - name: 'location'
+    type: String
+    description: |
+      The name of the location this data exchange subscription.
+    url_param_only: true
+    required: true
+    custom_flatten: 'templates/terraform/custom_flatten/bigquery_dataset_location.go.tmpl'
+    diff_suppress_func: 'tpgresource.CaseDiffSuppress'
+properties:
+  - name: 'destinationDataset'
+    type: NestedObject
+    required: true
+    ignore_read: true
+    description:
+      The destination dataset for this subscription.
+    properties:
+      - name: 'location'
+        type: String
+        description: |
+          The geographic location where the dataset should reside.
+          See https://cloud.google.com/bigquery/docs/locations for supported locations.
+        required: true
+        custom_flatten: 'templates/terraform/custom_flatten/bigquery_dataset_location.go.tmpl'
+        diff_suppress_func: 'tpgresource.CaseDiffSuppress'
+      - name: 'datasetReference'
+        type: NestedObject
+        required: true
+        description: A reference that identifies the destination dataset.
+        properties:
+          - name: 'datasetId'
+            type: String
+            description: A unique ID for this dataset, without the project name. The ID must contain only letters (a-z, A-Z), numbers (0-9), or underscores (_). The maximum length is 1,024 characters.
+            required: true
+          - name: 'projectId'
+            type: String
+            description: The ID of the project containing this dataset.
+            required: true
+      - name: 'friendlyName'
+        type: String
+        description: A descriptive name for the dataset.
+      - name: 'description'
+        type: String
+        description: A user-friendly description of the dataset.
+      - name: 'labels'
+        type: KeyValuePairs
+        description: |
+          The labels associated with this dataset. You can use these to
+          organize and group your datasets.
+  - name: 'name'
+    type: String
+    description: |-
+      The resource name of the subscription. e.g. "projects/myproject/locations/US/subscriptions/123"
+    output: true
+  - name: 'subscription_id'
```

Comment:
Some other property names are in camel case, should we be consistent?

Reply:
made them consistent